### ISSUE TITLE
[Feature Request] Background Blur for Bar

### ISSUE THREAD

**Issue by patrikzudel** [glzr-io/zebar#93]
### Describe the problem/motivation

Currently if we want to increase visibility of items on the bar, our options are limited when adjusting the background. 

### Describe the solution you'd like

It would be great if background blurring could be added, similar to how its done in Yasb. 
![image](https://github.com/user-attachments/assets/5d72d4a7-95d4-4c57-9b4f-5044b6ac7670)

**Comment by lars-berger** [comment]
Blur, mica, and acrylic are possible through the official [`window-vibrancy` plugin](https://github.com/tauri-apps/window-vibrancy)

**Comment by im-izz** [comment]
How did you get it to work. can you share dotfiles?

**Comment by im-izz** [comment]
@patrikzudel Can you share that config?

**Comment by lars-berger** [comment]
**Proposed config:**
```jsonc
  // Allowed values:  "blur", "acrylic", "mica", and "none".
  // On MacOS: "blur", "acrylic", and "mica" -> apply the MacOS vibrancy effect to the window.
  // On Linux: all effects instead add transparency to the window.
  "backgroundEffect": "blur",
```

LMK if anyone would like to take a stab at this 👍

**Comment by veryard** [comment]
Yeah - the effects Blur & Mica do not work that well on Windows 11 eg.
- Color isn't supported (https://docs.rs/tauri-utils/latest/x86_64-apple-darwin/tauri_utils/config/struct.WindowEffectsConfig.html#structfield.color)
- Blur: bad performance when resizing - Should be fine considering zebar should pin to top (https://github.com/tauri-apps/window-vibrancy?tab=readme-ov-file#available-functions)
- Acrylic: Same as above

I went ahead and made a PR to implement this, I do not have a Macbook so unfortunately I cannot test that part of the code.
